jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -267,14 +283,20 @@ String makeDockerfile() throws JsonProcessingException {
       dockerfile.append("\nEXPOSE ").append(port);
     }
 
-    boolean firstLabel = true;
+    StringJoiner joiner = new StringJoiner(" \\\n    ", "\nENV ", "");
+    for (Entry<String, String> env : environment.entrySet()) {
+      joiner.add(env.getKey() + "=" + objectMapper.writeValueAsString(env.getValue()));
+    }
+    if (environment.entrySet().size() > 0) {
+      dockerfile.append(joiner.toString());
+    }
+
+    joiner = new StringJoiner(" \\\n      ", "\nLABEL ", "");
```

Comment:
The logic for labels seems to be the same as the logic for environment - maybe condense to a static method?